docs/source/primitives.rst

```diff
@@ -286,6 +286,99 @@ Schema
 
 .. jsonschema:: ../../src/core/schema/components/component.json
 
+.. jsonschema:: ../../src/core/schema/common/component_lookup.json
+
+Button
+------
+
+.. jsonschema:: ../../src/core/schema/components/button.json
+
+Triggers action on click. The default buttons are `submit`, `next`, `back`. A next button can set a value in the context, allowing for branching of flows based on button presses.
+
+Actions
+^^^^^^^
+
+* `submit` moves to the next task while saving the contents of the form
+* `next` moves to the next task while ignoring form values
+* `back` moves to the previous screen
+
+.. .. code-block::
+..     {
+..         "type": "button",
+..         "action": "next",
+..         "style": "primary",
+..         "text": "Reset",
+..         "value": false,
+..         "destination_path": "$.save"
+..     }
+
+
+
+Data display
+------------
+
+Display data, this can take in a list of strings or objects containing label and value keys.
```

Comment:
how do the strings vs object with label and value keys use differ in use? From the description it isn't clear to me.